api/types/access_request.go

```diff
@@ -933,3 +935,37 @@ func ValidateAssumeStartTime(assumeStartTime time.Time, accessExpiry time.Time,
 
 	return nil
 }
+
+// RequestIDs is a collection of IDs for privilege escalation requests.
+type RequestIDs struct {
```

Comment:
What's the point of moving `RequestIDs` and `CertRoles` to `api/types`? Later changes?

Reply:
Was just trying to fix some cyclic imports.  In hindsight it didn't need to live in `api`.  I've moved this to `lib/sshca` and also switched most uses of this type to instead use `sshca.Identity` and/or `[]string` as appropriate.